aspnetcore/security/authentication/windowsauth.md

```diff
@@ -1,76 +1,65 @@
 ---
-title: Configure Windows Authentication in ASP.NET Core
+title: Configure Windows authentication in ASP.NET Core
 author: ardalis
-description: How to configure Windows Authentication in ASP.NET Core
-keywords: ASP.NET Core,
+description: This article describes how to configure Windows authentication in ASP.NET Core, using IIS Express, IIS, HTTP.sys, and WebListener.
```

Comment:
Http.Sys and WebListener? The versioning gets weird here huh?

Reply:
Until we get versioning in place for the aspnet/Docs repo, we're going to see weird things like this.